app/helpers.php

```diff
@@ -141,7 +141,13 @@ function es_search($params)
         $error = $e;
     }
 
-    Log::debug($error);
+    if (isset($error) && config('datadog-helper.enabled')) {
```

Comment:
`isset` check rather useless because it's always assigned with exception? And not sure why there's a null coalesce at the end =D

Reply:
oh wait, forgot about the return up there :best: